site/community.html

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Community
+id: community
+---
+
+<div class="sub-page">
+    <h3>Do you want to help build Contour?</h3>
```

Comment:
@amymanion this should be h1?

Reply:
@ccbayer There already is an h1, but it should be an h2. Updated it.